styles.css

```diff
@@ -472,6 +476,79 @@ h1::after {
   margin-top: 10px;
 }
 
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  padding-bottom: 2rem;
+  margin-top: 4578px;
+}
+
+.contact-option-lists {
+  width: 327px;
+  height: 216px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 54px;
+  margin-top: 73px;
+}
+
+.contact-form li {
+  list-style: none;
+}
+
+input {
+  width: 295px;
+  height: 18px;
+  padding-top: 1.5rem;
+  padding-bottom: 1.5rem;
+  resize: none;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  margin-left: -14px;
+  margin-right: -14px;
+}
+
+textarea {
+  height: 114px;
+  width: 295px;
+  border-bottom: 1px solid #dbd8d7;
+  resize: none;
+  background: #fbf8f7;
+  margin-left: -14px;
+  margin-right: 24px;
+  margin-top: 39px;
+}
+
+.btn-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 131px;
+  height: 48px;
+  margin-left: 122px;
+  margin-top: 5px;
+  margin-right: 122px;
+  background: #ff6b00;
+}
+
+.btn-btn:hover {
+  border-color: blue;
+}
```

Comment:
- [ ] Please add the button interaction styling for the submit button. Make sure the text and background color is the same as the Figma design. 

![Buttons(3)](https://user-images.githubusercontent.com/12524453/177429422-b52470e6-01e1-468d-a2bd-d07924474232.png)